kolibri/core/assets/src/core-app/urls.js

```diff
@@ -28,6 +33,13 @@ const urls = {
     const urlObject = new URL(url, base);
     return urlObject.href;
   },
+  storageUrl(fileId, extension, embeddedFilePath = '') {
+    const filename = `${fileId}.${extension}`;
+    if (['perseus', 'zip', 'h5p'].includes(extension)) {
+      return this['kolibri:core:zipcontent'](filename, embeddedFilePath);
+    }
+    return generateUrl(this.__contentURL, `${filename[0]}/${filename[1]}/${filename}`);
```

Comment:
might just reference `getPluginData().urls.__contentURL` directly? Copying it to `this` seems unnecessary

Reply:
Fair.

Reply:
It just came long for free with the other Urls during the `Object.assign` which does need to happen to expose the functions directly on the `urls` export.